src/main/java/com/powsybl/openloadflow/network/VoltageControl.java

```diff
@@ -63,6 +66,22 @@ public boolean isControllerEnabled(T controllerElement) {
         throw new IllegalStateException();
     }
 
+    public static List<Integer> getVoltageTargetPriority() {
+        return voltageTargetPriority;
+    }
+
+    public static void setVoltageTargetPriority(List<Integer> voltageTargetPriority) {
```

Comment:
Just for my understanding: why all of this is not done only at network loading when we build all the voltage controls? These priorities are changing during computations? Because if could be stored in Generator/Shunt/TransformerVoltageControl, no?

Reply:
I have done it at network loading. Priorities are not changed during computations. I choose to store them in LfNetwork to do not duplicate them in every VoltageControl.
See https://github.com/powsybl/powsybl-open-loadflow/pull/955#discussion_r1474721124

Reply:
To be discuss, but I thought it could be a dedicated attribute in each control type. @jeandemanged what do you think ?

Reply:
I agree, this was moved to `VoltageControl` in 982ec958a966797b5e9f65548508d58bd7ed2ba2